### PR TITLE
feat(tooltips): add tooltip component

### PR DESCRIPTION
**Pull request by rossmoody**
## Description

This PR adds the Tooltip component to the website. 

## Checklist

- [ ] :ok_hand: website updates are Garden Designer approved (add the designer as a reviewer)
- [ ] :black_nib: copy updates are approved (add the content strategist as a reviewer)
- [ ] :link: considered opportunities for adding cross-reference URLs (grep for keywords)
- [ ] :wheelchair: analyzed via [axe](https://www.deque.com/axe/) and evaluated using VoiceOver
- [ ] :zap: audited via [web.dev](https://web.dev/measure/) for performance and SEO
- [ ] :memo: tested in Chrome, Firefox, Safari, Edge, and IE11

### CODE REVIEW

File: src/pages/components/tooltip.mdx [hunk 1]
```diff
@@ -0,0 +1,95 @@
+---
+title: Tooltip
+description: >
+  Tooltips appear when a user hovers over an element. They provide contextual
```
Comment (jzempel) [inline src/pages/components/tooltip.mdx:4]: They also appear on focus.

File: src/pages/components/tooltip.mdx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  information about the element they are paired with.
+package: '@zendeskgarden/react-tooltips'
+components:
+  - tooltips/src/elements/Tooltip.tsx
```
Comment (jzempel) [inline src/pages/components/tooltip.mdx:8]: Add

```
- tooltips/src/elements/Paragraph.tsx
- tooltips/src/elements/Title.tsx
- tooltips/src/elements/Tooltip.tsx
```

File: src/pages/components/tooltip.mdx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+## API
+
+### Tooltip
+
+<PropSheet components={props.data.mdx.components} componentName="tooltip" />
+
+#### Paragraph
+
+#### Title
```
Comment (jzempel) [inline src/pages/components/tooltip.mdx:87]: Both `Paragraph` and `Title` are first-class citizens in this package. Set to `H3`, alphabetize, and add associated `<PropSheet>` components.

**Review by jzempel — CHANGES_REQUESTED** [review]
Per https://brandland.zendesk.com/copywriting#technical-rules should "centre" be replaced with "center"? (Sorry, UK.)

File: src/examples/tooltip/TooltipPlacement.tsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  bottomEnd: 'bottom-end',
+  start: 'start',
+  startTop: 'start-top',
+  startBottom: 'start-bottom'
+};
+
+const Example = () => (
+  <Row style={{ margin: 80 }}>
+    <Col textAlign="center">
+      <Tooltip placement={PLACEMENTS.topStart} content={'Eat, drink and be rosemary'}>
```
[the remark this answers is not in the document]
Comment (rossmoody) [reply src/examples/tooltip/TooltipPlacement.tsx:32]: Yeah, for this one I followed the Menu implementation pattern for arrow positioning (which I also thought was there for reference).

File: src/pages/components/tooltip.mdx [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+## Configuration
+
+<Configuration reactPackage={props.data.mdx.package} components={props.data.mdx.components} />
+
+## API
+
+### Paragraph
+
+### Title
```
Comment (jzempel) [inline src/pages/components/tooltip.mdx:85]: Please add `<PropSheet>` components for these guys. Even if they render empty now, they'll be here to pick up potential future props.

**Review by jzempel — APPROVED** [review]

File: src/pages/components/tooltip.mdx (hunk 5 above)
Comment (jzempel) [reply src/pages/components/tooltip.mdx:85]: Or does ☝️  end up showing empty table headings? Hopefully not. I _think_ that got fixed.

Comment (rossmoody) [reply src/pages/components/tooltip.mdx]: Looks like it still renders blank. Would we want to do this for Grid and Chrome by any chance? I was omitting them if they didn't have props to account for.

Comment (jzempel) [reply src/pages/components/tooltip.mdx]: Yes, please throw them in. It gives us options for seeing future props or flowing more data through if we choose to (ex. you and I talked about pulling in an `extends` clause).

**Pull request merged**